package.json

```diff
@@ -21,13 +21,15 @@
   "license": "MIT",
   "scripts": {
     "lint": "vue-cli-service lint",
+    "lint:ci": "vue-cli-service lint --no-fix",
     "build:lib": "vue-cli-service build --target lib --name homeday-blocks ./src/components/index.js",
     "build:storybook": "vue-cli-service storybook:build -c config/storybook",
     "serve:storybook": "vue-cli-service storybook:serve -p 6006 -c config/storybook",
     "test:unit": "vue-cli-service test:unit",
     "test:unit:snapshot": "vue-cli-service test:unit -u",
     "test:unit:watch": "npm run test:unit -- --watch",
     "test:unit:ci": "npm run test:unit -- --ci --coverage",
+    "test:percy": "if [ $PERCY_TOKEN ]; then percy-storybook; else echo 'Skipping test: PERCY_TOKEN is not defined'; fi",
```

Comment:
We need to check for `PERCY_TOKEN` otherwise it breaks as the variable can't be found.